src/store/profileSlice.js

```diff
@@ -0,0 +1,109 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { STATUSES } from '../constants/statuses';
+import { getCookie } from '../utils/cookie';
+
+const initialState = {
+  profile: null,
+  status: STATUSES.IDLE,
```

Comment:
rename `profile` to `data` for consistency and add an `errors` array to use in toasts
```suggestion
  data: null,
  status: STATUSES.IDLE,
  errors: []
```

Reply:
I am handling it in other PR, where i am working on toasts